packages/components/tooltip/src/tooltip.vue

```diff
@@ -1,3 +1,177 @@
+<script setup lang="ts">
```

Comment:
Please revert the order of <script> and <template>. It's hard to review in the one PR. We decide to change the order in the separate PR.